charts/kubeclarity/templates/scanner-template-configmap.yaml

```diff
@@ -99,6 +99,13 @@ data:
             - name: SCANNER_DEPENDENCY_TRACK_API_KEY
               value: {{ index .Values "kubeclarity-runtime-scan" "vulnerability-scanner" "scanner" "dependency-track" "apiKey" }}
 {{- end }}
+{{- $noproxy := list (print (include "kubeclarity.name" .) "." .Release.Namespace ":" (index .Values "kubeclarity-runtime-scan" "resultServicePort")) -}}
+{{- if index .Values "kubeclarity-grype-server" "enabled" -}}
+{{- $noproxy = append $noproxy (print (include "kubeclarity.grype-server.name" .) "." .Release.Namespace ":" (index .Values "kubeclarity-grype-server" "servicePort")) -}}
+{{- end -}}
+{{- if index .Values "kubeclarity-trivy-server" "enabled" -}}
+{{- $noproxy = append $noproxy (print ((index .Values "kubeclarity-trivy-server" "service" "name") | default (include "trivy.fullname" (index .Subcharts "kubeclarity-trivy-server"))) "." .Release.Namespace ":" (index .Values "kubeclarity-trivy-server" "service" "port")) -}}
+{{- end -}}
```

Comment:
Generally its good practise to define variables at the top of the template just to prevent them affecting indentation and also to keep the template as readable as possible. There are a few other templates in the chart where you can find this being done. Would you mind moving this up please.

Reply:
done